packages/apps/MintingB2B/src/pages/NFTDetails/index.tsx

```diff
@@ -184,46 +209,39 @@ const WalletPage = () => {
-                          normalData.status === "PRE_STAGE" && (
-                            <Grid columns={2} gap={16}>
-                              <Button btnType="filled" style={{ width: "100%" }}>
-                                Edit
-                              </Button>
-                              <Button
-                                btnType="outlined"
-                                style={{ width: "100%" }}
-                              >
-                                Stage
-                              </Button>
-                            </Grid>  
-                          )
-                        }
-                        {
-                          normalData.status === "WAITING_FOR_OWNER" && (
-                            <Grid columns={2} gap={16}>
-                              <Button onClick={generateQR} btnType="filled" style={{ width: "100%" }}>
-                                Generate QR
-                              </Button>
-                              <Button
-                                onClick={() => setIsOpen(true)}
-                                btnType="outlined"
-                                disabled={normalData.status !== 'WAITING_FOR_OWNER'}
-                                style={{ width: "100%" }}
-                              >
-                                Connect with QR
-                              </Button>
-                              <Button
-                                onClick={() => setIsSendOpen(true)}
-                                btnType="outlined"
-                                disabled={normalData.status !== 'WAITING_FOR_OWNER'}
-                                style={{ width: "100%" }}
-                              >
-                                Send to Principal
-                              </Button>
-                            </Grid>
-                          )
-                        }
+                        {normalData.status === 'PRE_STAGE' && (
+                          <Grid columns={2} gap={16}>
+                            <Button btnType="filled" style={{ width: '100%' }}>
+                              Edit
+                            </Button>
+                            <Button btnType="outlined" style={{ width: '100%' }}>
+                              Stage
+                            </Button>
+                          </Grid>
+                        )}
+                        {normalData.status === 'WAITING_FOR_OWNER' && (
+                          <Grid columns={2} gap={16}>
+                            <Button onClick={generateQR} btnType="filled" style={{ width: '100%' }}>
+                              Generate QR
+                            </Button>
+                            <Button
+                              onClick={() => setIsOpen(true)}
+                              btnType="outlined"
+                              disabled={normalData.status !== 'WAITING_FOR_OWNER'}
```

Comment:
Please replace any repeating status strings with a constant?

Reply:
Strings changed to const and created the switch statement for the NFT Status